packages/ng/libraries/root/src/style/definitions/option/_option-searcher.scss

```diff
@@ -14,8 +14,18 @@
 	.textfield-input {
 		border-radius: 0;
 		box-shadow: none;
+		background: #FFFFFF;
```

Comment:
background or background-color (look, line 25), that is the question

Reply:
replace occurences of `#FFFFFF` with the recently added `_theme("colors.white");` ?

Reply:
i just checked and the colors you are refering to dont exist in the theme

https://github.com/LuccaSA/lucca-front/blob/rc/packages/scss/src/theming/_palettes.scss

is it an oversight ? should it be added to the "standard" color map ?